server/src/test/integration/org/apache/cassandra/sidecar/acl/MutualTLSAuthenticationIntegrationTest.java

```diff
@@ -146,4 +222,10 @@ private void grantSidecarPermission(String role, String resource, String permiss
         session.execute(String.format("INSERT INTO sidecar_internal.role_permissions_v1 (role, resource, permissions) " +
                                       "VALUES ('%s', '%s', {'%s'})", role, resource, permission));
     }
+
+    private void grantRole(String parentRole, String childRole)
```

Comment:
Nit: Is this parentRole childRole? I kind of find the naming confusing. What about roleName, roleToAssign?